translations/generateLanguageFiles.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+MODULE_NAMES=${1:-"jaspTestModule"}
+PATH_TO_R_FILE=$2
+LANGUAGE_CODES=${3:-"nl de es pt ja tr"}
```

Comment:
I am still not sure how this will work: will this file be copied to all submodules? In this case the list of language codes must come from a centralized place.

Reply:
this file will only stay in this repo, and the language codes can be adjusted in one place (this repo).

Reply:
this [yaml file](https://github.com/vandenman/jaspTestModule/blob/master/.github/workflows/translations.yml), probably with some minor adjustments, would need to be copied to each repository.